dbbackup/management/commands/mediabackup.py

```diff
@@ -95,6 +90,11 @@ def get_source_dir(self):
 
     def create_backup_file(self, source_dir, backup_basename, **kwargs):
         temp_dir = tempfile.mkdtemp(dir=dbbackup_settings.TMP_DIR)
+        # Amazon S3 data model is a flat structure. However, Amazon Console
```

Comment:
```
Traceback (most recent call last):
  File "cscsite/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/pacahon/.pyenv/versions/csc/lib/python2.7/site-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "/Users/pacahon/.pyenv/versions/csc/lib/python2.7/site-packages/django/core/management/__init__.py", line 330, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/pacahon/.pyenv/versions/csc/lib/python2.7/site-packages/django/core/management/base.py", line 393, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/pacahon/.pyenv/versions/csc/lib/python2.7/site-packages/django/core/management/base.py", line 444, in execute
    output = self.handle(*args, **options)
  File "/Users/pacahon/Downloads/tmp/django-dbbackup/dbbackup/utils.py", line 88, in wrapper
    func(*args, **kwargs)
  File "/Users/pacahon/Downloads/tmp/django-dbbackup/dbbackup/management/commands/mediabackup.py", line 43, in handle
    options.get('no_compress')^True)
  File "/Users/pacahon/Downloads/tmp/django-dbbackup/dbbackup/management/commands/mediabackup.py", line 63, in backup_mediafiles
    compress=compress
  File "/Users/pacahon/Downloads/tmp/django-dbbackup/dbbackup/management/commands/mediabackup.py", line 105, in create_backup_file
    if kwargs.get('compress') \
  File "/Users/pacahon/.pyenv/versions/2.7.9/lib/python2.7/tarfile.py", line 1694, in open
    stream = _Stream(name, filemode, comptype, fileobj, bufsize)
  File "/Users/pacahon/.pyenv/versions/2.7.9/lib/python2.7/tarfile.py", line 402, in __init__
    fileobj = _LowLevelFile(name, mode)
  File "/Users/pacahon/.pyenv/versions/2.7.9/lib/python2.7/tarfile.py", line 375, in __init__
    self.fd = os.open(name, mode, 0666)
OSError: [Errno 2] No such file or directory: '/tmp/tmpply45e/UNIT-739.local-2015-08-31/media.tar.gz'
```
For AWS storage `/UNIT-739.local-2015-08-31/media.tar.gz` - 1 object. For my filesystem - dir + file and `UNIT-739.local-2015-08-31` doesn't exists.